src/vunnel/providers/bitnami/parser.py

```diff
@@ -0,0 +1,101 @@
+from __future__ import annotations
+
+import logging
+import os
+from typing import TYPE_CHECKING
+
+import orjson
+
+if TYPE_CHECKING:
+    from vunnel.workspace import Workspace
+
+from .git import GitWrapper
+
+namespace = "bitnami"
+
+
+class Parser:
+    _git_src_url_ = "https://github.com/bitnami/vulndb.git"
+    _git_src_branch_ = "main"
+
+    def __init__(self, ws: Workspace, logger: logging.Logger | None = None):
+        self.workspace = ws
+        self.git_url = self._git_src_url_
+        self.git_branch = self._git_src_branch_
+        self.urls = [self.git_url]
+        if not logger:
+            logger = logging.getLogger(self.__class__.__name__)
+        self.logger = logger
+        _checkout_dst_ = os.path.join(self.workspace.input_path, "vulndb")
+        self.git_wrapper = GitWrapper(
+            source=self.git_url,
+            branch=self.git_branch,
+            checkout_dest=_checkout_dst_,
+            logger=self.logger,
+        )
+
+    def _load(self):
+        self.logger.info("loading data from git repository")
+
+        vuln_data_dir = os.path.join(self.workspace.input_path, "vulndb", "data")
+        for root, dirs, files in os.walk(vuln_data_dir):
+            dirs.sort()
+            for file in sorted(files):
+                full_path = os.path.join(root, file)
+                with open(full_path, encoding="utf-8") as f:
+                    yield orjson.loads(f.read())
+
+    def _normalize(self, vuln_entry):
+        self.logger.info("normalizing vulnerability data")
+
+        vuln_id = vuln_entry["id"]
+        if "aliases" in vuln_entry and len(vuln_entry["aliases"]) > 0:
+            vuln_id = vuln_entry["aliases"][0]
+        fixed_in = []
+        if "affected" in vuln_entry:
+            for affected in vuln_entry["affected"]:
+                version = "None"
+                if "ranges" in affected:
+                    for r in affected["ranges"]:
+                        if "events" in r:
+                            for event in r["events"]:
+                                # TODO: manage last_affected
+                                # if events["last_affected"]:
+                                #     version = events["last_affected"]
+                                #     break
+                                if "fixed" in event:
+                                    version = event["fixed"]
+                                    break
+
+                fixed_in.append(
+                    {
+                        "Name": affected["package"]["name"],
+                        "VersionFormat": "semver",
+                        "NamespaceName": namespace,
+                        "Version": version,
+                    },
+                )
+        link = "None"
+        if "references" in vuln_entry and len(vuln_entry["references"]) > 0:
+            link = vuln_entry["references"][0]
+
+        return vuln_id, {
+            "Vulnerability": {
```

Comment:
I think we want the second return value here to conform to the OSV schema.

I'm simplifying a bit, but second element of the returned tuple will be JSON-serialized and eventually picked up by the anchore/grype-db#217 (and some follow up work) and transformed into the Grype-specific vulnerability schema that Grype matches on.

This dict seems to imitate the shape of some dictionaries returned by other parsers, which looks right, but in this case I think we want a real OSV record.

I'm going to try to do some refactoring to make it more obvious what things `parser.get()` should return.